python-clusters/attach-aks-cluster/cluster.json

```diff
@@ -9,24 +9,81 @@
 
     "params": [
         {
-            "name": "connectionInfo",
+            "name": "connectionInfoV2",
             "label" : "Connection",
             "type": "PRESET",
-            "parameterSetId" : "connection-info",
+            "parameterSetId" : "connection-info-v2",
             "mandatory" : true
         },
+        {
+            "name": "useSameNameAsDSSCluster",
```

Comment:
why add a boolean? Just saying "value in clusterName, if empty default to DSS cluster name" would be fine, no? (basically, why not just use the "cluster" field like before?)

Reply:
Same goes for resource group and location. The rationale is usability, as a checked field is more quickly understood than a empty field.